R/utils.R

```diff
@@ -173,18 +183,187 @@ select_base_url <- function(server) {
   return(base_url)
 }
 
+
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Formatting functions -------------
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+
+#' rename columns in dataframe
+#'
+#' @param df  data frame
+#' @param oldnames  character: old names
+#' @param newnames  character: new names
+#'
+#' @return data frame with new names
+#' @keywords internal
+renamecols <- function(df, oldnames, newnames) {
```

Comment:
Change the name to `rename_cols` for consistency with other functions in the package ?

Reply:
Yes. Good idea.